GNUmakefile

```diff
@@ -121,3 +121,8 @@ scaffold:
 	@go run ./tools/scaffold/*.go $(name) $(type)
 	@echo "Reminder: configure the new $(type) in provider.go"
 
+
+.PHONY: generate-doc
+generate-doc: ## Generate the resource documentation via tfplugindocs
```

Comment:
is this generated on demand?   or we want to have something similar to test mocks to check if the user forgot to run the command by running it in a GH action and checking that there are not file changes.

Reply:
Good point, this can be useful to ensure docs and schema descriptions/changes are kept synced. This would however require an additional command that calls the docs generation for all resources which have documentation autogenerated.

Reply:
I agree that this is a good idea. I believe it is still early to implement it as it should require moving all the resources to the auto-generation of the doc so that we don't have to keep track of what resources are using the gen tool and which are not. I will create a ticket to track this work once all the resources are migrated